components/const.py

```diff
@@ -227,3 +227,4 @@
     "freelancers. An example of those would be https://fiverr.com, which we are not "
     "associated with."
 )
+TOKEN_TEXT = "⚠️ You posted a token, go revoke it with @BotFather.\n\n"
```

Comment:
```suggestion
TOKEN_TEXT = "⚠️ You posted a token, go revoke it with @BotFather.\n\n"
```

phrasing suggesiton: `You posted the token(s) of the following bots: @username1, @username2, … and others (in case there's more than 10). Go revoke them with @Botfather\n\n. Days since last token was shared: …`